### ISSUE TITLE
psf to take over request/toolbelt project?

### ISSUE THREAD

**Issue by RosterIn** [psf/requests#6219]
requests project was moved to psf
however there was another project toolbelt https://github.com/requests/toolbelt which is not maintained.
according to https://github.com/requests/toolbelt/issues/322 it's suggested that psf might transfer ownership as it make sense that `requests-toolbelt `will be with same ownership as `requests`

What is psf take on this?

**Comment by wqh17101** [comment]
Agree. Or merge it into requests
Such as the ability below which is used by lots of users.
```
If you are sending a very large file as a multipart/form-datarequest , you may want to stream the request. requestsNot supported by default , but a third-party package is supported. You can read [the toolbelt](https://toolbelt.rtfd.org/)[ documentation](https://toolbelt.rtfd.org/) to learn how to use it.requests-toolbelt
```

**Comment by sigmavirus24** [comment]
No one on that thread is involved with the project or the PSF

**Comment by wqh17101** [comment]
@sigmavirus24 How about merge it and remove https://github.com/requests/toolbelt

**Comment by sigmavirus24** [comment]
This library is under feature freeze so merging it would violate that. Additionally a lot of things in the toolbelt are niché use cases or proofs if concept for others to build on top of, not things that belong in requests. As it is the streaming multipart form data encoder is being moved into urllib3 to be more generally useful. I believe that's what 99% of toolbelt users need it for anyway.

**Comment by RosterIn** [comment]
The thing is that there is no alternative for toolbelt. This is why i asked here if psf has intentions of taking over this sub-project too.

**Comment by sigmavirus24** [comment]
> This is why i asked here if psf has intentions of taking over this sub-project too

It has never been a sub-project. It's where I started collecting recipes for things people were often wanting but which don't belong in this project. It's never been a particularly high priority for me.

I also don't think people understand that moving organizations in GitHub doesn't change maintenance or the people involved (presently just me). The psf doesn't magic contributors who provide patches or people to help maintain.

**Comment by achapkowski** [comment]
@sigmavirus24 I understand your point about maintaining the API. The toolbelt might get more help if it's moved into the `psf` org. 

Just my thought, feel free to ignore as needed. :)

**Comment by sigmavirus24** [comment]
I can add people to the requests org if people care about the project but it sounds like people care more about joining the psf org and not the project. To be clear, the largest code of any value in the toolbelt will be in urllib3 v2.0. I'm not convinced the effort of moving the project is worth anything.

**Comment by pquentin** [comment]
@sigmavirus24 Speaking as a fellow urllib3 maintainer here, I know we need to get your PR merged for urllib3 v2.0 but in the meantime I would be happy to have maintainers right on the toolbelt repo and the PyPI package to fix the urllib3 warnings that it's currently emitting. I agree that moving to the psf org isn't needed and to be clear I'm also not promising to fix any other issues.

What do you think?

**Comment by sigmavirus24** [comment]
I'll add you as soon as I'm able @pquentin

**Comment by achapkowski** [comment]
I'll help maintain the package as well.

**Comment by pquentin** [comment]
Thank you @sigmavirus24!

@achapkowski I'm sure help will be needed to review PRs, fix CI, support new Python versions, etc.

**Comment by achapkowski** [comment]
@pquentin cool, send stuff my way!

**Comment by pquentin** [comment]
With help from @achapkowski and @sigmavirus24 I released requests-toolbelt 0.10.0 today. There is no new work in it, it only contains all the fixes that were merged since 0.9.1, as well as a new feature, hence the bump to 0.10.0.

I think this issue can now be closed.
